File: Trial/show.php
<?php
$servername = "localhost";
$username = "root";
$password = "";
$dbname = "book";


// Create connection
$conn = mysqli_connect($servername, $username, $password, $dbname);
// Check connection
if (!$conn) {
  die("Connection failed: " . mysqli_connect_error());
}

$sql = "SELECT * from book";
$result = mysqli_query($conn, $sql);
echo"<center><h1><u>Book Information</u></h1></center>";
echo "<center><table border='3'><tr><td>BookID</td><td>Book Name</td><td>Author</td><td>Publisher</td><td>Price</td></tr>";
if (mysqli_num_rows($result) > 0) {
  // output data of each row
  while($row = mysqli_fetch_assoc($result)) {
    echo "<tr><td>" . $row["Bookid"]. "</td><td>" . $row["Name"]. "</td><td> " . $row["Author"]."</td><td> " . $row["Publisher"]." </td><td>". $row["Price"]."</td></tr>";
  }
  echo"</table></center>";
} else {
  echo "0 results";
}

mysqli_close($conn);
?>
		<a href="bookstore.html">Back</a>

File: PythonProject/show.py
#!C:/Python36/python.exe
print("Content-Type: text/html\n\n")
print()

import cgi
import pymysql
con=pymysql.connect('localhost','root','','test')
cur=con.cursor()
cur.execute("SELECT * FROM BOOKS")
print("<h3>Book Details</h3>")
print("<table border='1'><tr><th>BookID</th><th>Title</th><th>Author</th><th>Publisher</th></tr>")
result=cur.fetchall()
for row in result:
	bookid=row[0]
	title=row[1]
	author=row[2]
	publisher=row[3]
	print("<tr><td>")
	print(bookid)
	print("</td>")
	print("<td>")
	print(title)
	print("</td>")
	print("<td>")
	print(author)
	print("</td>")
	print("<td>")
	print(publisher)
	print("</td></tr>")
print("</table>")
print("<a href='http://localhost/PythonProject/a.html'> Back</a>")
con.commit()
cur.close()